src/backend/metal/src/command.rs

```diff
@@ -1373,6 +1393,18 @@ impl RawCommandQueue<Backend> for CommandQueue {
 
         command_buffer.commit();
 
+        if let Some(ref mut wait) = self.frame_wait {
+            wait.frame += 1;
+            if wait.frame >= FRAME_WAIT_REPORT_WINDOW {
+                let time = wait.duration / wait.frame as u32;
+                println!("Frame wait time={}ms count={}",
```

Comment:
The `println` could be undesirable for some applications if they're ok with this